### PR TITLE
fixed tests connectors buttons with a sandwich menu #2403

### PR DESCRIPTION
**Pull request by David-DB88**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

**What changes did you make?** (Give an overview)
Replaced connectors buttons with a sandwich menu,
Fixed tests

**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [ ] No need to
- [x] Manually (please, describe, if necessary)
- [ ] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: kafka-ui-react-app/src/components/Connect/Details/Actions/Actions.tsx [hunk 1]
```diff
@@ -63,66 +63,30 @@ const Actions: React.FC = () => {
...
+        </DropdownItem>
+        <DropdownItem onClick={restartAllTasksHandler} disabled={isMutating}>
+          Restart All Tasks
+        </DropdownItem>
+        <DropdownItem onClick={restartFailedTasksHandler} disabled={isMutating}>
+          Restart Failed Tasks
+        </DropdownItem>
+        <DropdownItem onClick={deleteConnectorHandler} disabled={isMutating}>
+          Delete
+        </DropdownItem>
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Connect/Details/Actions/Actions.tsx:88]: Should be `red`

File: kafka-ui-react-app/src/components/Connect/Details/Actions/__tests__/Actions.spec.tsx [hunk 2]
```diff
@@ -122,9 +128,7 @@ describe('Actions', () => {
           mutateAsync: restartConnector,
         }));
         renderComponent();
-        userEvent.click(
-          screen.getByRole('button', { name: 'Restart Connector' })
-        );
+        userEvent.click(screen.getByText('Restart Connector'));
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Connect/Details/Actions/__tests__/Actions.spec.tsx:131]:
```suggestion
        userEvent.click(
          screen.getByRole('menuitem', { name: 'Restart Connector' })
        );
```

**Review by workshur — COMMENTED** [review]
Looks good so far

**Review by workshur — APPROVED** [review]

**Comment by workshur** [comment]
@Haarolean pls take a look 👀

**Review by Haarolean — APPROVED** [review]

**Comment by Haarolean** [comment]
@provectus/kafka-qa @VladSenyuta got one test broken :(

**Comment by VladSenyuta** [comment]
> @provectus/kafka-qa @VladSenyuta got one test broken :(

fixed

**Review by VladSenyuta — APPROVED** [review]

**Pull request merged**